inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/LoadNode.java

```diff
@@ -60,7 +62,9 @@
         @JsonSubTypes.Type(value = ClickHouseLoadNode.class, name = "clickHouseLoad"),
         @JsonSubTypes.Type(value = SqlServerLoadNode.class, name = "sqlserverLoad"),
         @JsonSubTypes.Type(value = TDSQLPostgresLoadNode.class, name = "tdsqlPostgresLoad"),
-        @JsonSubTypes.Type(value = MySqlLoadNode.class, name = "mysqlLoad")
+        @JsonSubTypes.Type(value = MySqlLoadNode.class, name = "mysqlLoad"),
+        @JsonSubTypes.Type(value = IcebergLoadNode.class, name = "icebergLoad"),
+        @JsonSubTypes.Type(value = ElasticsearchLoadNode.class, name = "elasticsearchLoad")
```

Comment:
`Node` add @JsonSubTypes.Type(value = ElasticsearchLoadNode.class, name = "elasticsearchLoad"). And a UT in inlong-sort/sort-common/src/test/java/org/apache/inlong/sort/protocol/node/load.